README.md

```diff
@@ -98,149 +98,142 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Added enable and disable telemetry to align it with the other utilities.
 
 ### Command Not Found
 
- - Signed the PowerShell scripts used by the Command Not Found installation process.
+ - Added telemetry for when a module instance is created in PowerShell.
+
+### FancyZones
+
+ - Fixed a memory leak occurring on work area changes.
 
 ### File Explorer add-ons
 
- - Fixed an issue causing SVG Thumbnail generation to hang when trying to preview SVG files at the same time.
+ - Added support to the .ksh, .zsh, .bsh and .env file types to Monaco previewer. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+ - Re-enabled the RendererAppContainer feature in WebView 2, since the associated crash has been fixed in the latest WebView 2 releases.
 
 ### File Locksmith
 
- - Improved the context menu entry caption. Thanks [@niels9001](https://github.com/niels9001)!
+ - Added a Windows 11 tier 1 context menu.
 
-### Find My Mouse
+### Hosts File Editor
 
- - Added more settings to tune shake detection when activating through mouse shake.
+ - Tweaked filter button style to indicate if filters are applied.
+ - Added an error indicator to each input field to indicate why a new entry can't be created.
+ - Added a in-line delete button for each entry.
 
-### Hosts File Editor
+### Image Resizer
 
- - Added a feature to duplicate an entry. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Units and resize modes are now localized.
+ - Fixed an issue causing the Height combo box to not hide when selecting percent as the measure unit in the Fit and Fill resize modes.
+ - Tweaked and improved UI. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
 
-### Installer
+### Keyboard Manager
 
- - Included the new languages localization files in the installer.
+ - Added a feature that allows remapping a shortcut to starting an application. Thanks [@jefflord](https://github.com/jefflord)!
+ - Added a feature that allows remapping a shortcut to open a URI. Thanks [@jefflord](https://github.com/jefflord)!
+ - Added chords to shortcuts. Thanks [@jefflord](https://github.com/jefflord)!
+ - Send telemetry about the key/shortcut to key/shortcut remappings that are set. This doesn't include remap to text, application or URI since those might contain personal information.
+ - Added telemetry to send a daily event that at least a key/shortcut to key/shortcut remapping was used.
+ - Tweaked and fixed the chords code to better follow conventions when trying to call the same chord multiple times.
 
-### Image Resizer
+### Mouse Without Borders
+ - Fixed an issue causing the target path string to be corrupted when registering as a service.
+
+### Paste as Plain Text
 
- - Improved the context menu entry caption. Thanks [@niels9001](https://github.com/niels9001)!
+ - Prevent the windows menu from activating when the windows key is part of the activation shortcut and is released sooner.
```

Comment:
```suggestion
 - Prevent the start menu from activating when the windows key is part of the activation shortcut and is released sooner.
```

Reply:
Caps

Reply:
I've addressed the feedback in https://github.com/microsoft/PowerToys/pull/31648/commits/b3b0ee3f9c6233f9b79a9dbe0b9397cbab7a5136. Thank you!